src/live_restore/live_restore_fs.c

```diff
@@ -886,17 +885,28 @@ __wti_live_restore_fs_fill_holes(WT_FILE_HANDLE *fh, WT_SESSION *wt_session)
     WT_TIMER timer;
     uint64_t msg_count = 0;
     bool finished = false;
+    char *buf = NULL;
+    WT_SESSION_IMPL *session = (WT_SESSION_IMPL *)wt_session;
+    WT_RET(
+      __wt_malloc(session, ((WTI_LIVE_RESTORE_FS *)S2C(session)->file_system)->read_size, &buf));
```

Comment:
Do you want `wt_calloc` to make sure the buffer is cleared after allocation?

Reply:
042d20ceb